misc/package/Data Files/MWSE/core/mcm/components/settings/KeyBinder.lua

```diff
@@ -52,7 +53,8 @@ function KeyBinder:getComboString(keyCombo)
 	                 tes3.scanCode.rShift)
 	local hasCtrl = (keyCombo.isControlDown and keyCombo.keyCode ~= tes3.scanCode.lCtrl and keyCombo.keyCode ~=
 	                tes3.scanCode.rCtrl)
-	local prefix = (hasAlt and "Alt-" or hasShift and "Shift-" or hasCtrl and "Ctrl-" or "")
+	local hasSuper = (keyCombo.isSuperDown and keyCombo.keyCode ~= tes3.scanCode.lSuper)
```

Comment:
Before we merge this, I need to find out the scan code for the right super key.

Reply:
Right, I found the scancodes for the right windows key. It's in https://github.com/MWSE/MWSE/pull/429, so that should be merged before this.